storage/client/client.go

```diff
@@ -41,7 +41,7 @@ type StorageClient struct {
 // private method found at https://github.com/jackc/pgtype/blob/master/numeric.go#L398
 func (c *StorageClient) numericToBigInt(ctx context.Context, n *pgtype.Numeric) (BigInt, error) {
 	if n.Exp == 0 {
-		return BigInt{*n.Int}, nil
+		return BigInt{Int: *n.Int}, nil
```

Comment:
The linter started complaining about these. I don't know why; maybe it has to do with moving it to a different package (and the previous package had some check disabled), or maybe we use a non-pinned linter version and it happened to change while I was working on this PR. I was lazy and just made the linter happy this rather than finding the root cause.
cc @pro-wh since you asked.

Reply:
oh interesting